src/module_TOUCHSCREEN/TouchscreenBackEnd.java

```diff
@@ -6,7 +6,7 @@
  * 
  */
 
-package module_TOUCHSCREEN;
```

Comment:
If this entire class if just commented out, do we really even need it right now? We could just delete it.